docs/packages/Parser.mdx

```diff
@@ -421,16 +421,18 @@ This option allows you to toggle the warning that the parser will give you if yo
 ```plaintext
 =============
 
-WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.
+WARNING: @typescript-eslint/typescript-estree does not support your TypeScript version.
```

Comment:
-1 from me on this change:

* Starting with an org-scoped package name makes people less likely to read it.
* The _"officially"_ note is important: most of the time, new TS versions mostly "just work" with not-yet-there parser versions. We just don't guarantee 100% workability.